apps/backoffice-v2/src/pages/Entities/Entities.page.tsx

```diff
@@ -44,13 +45,18 @@ export const Entities: FunctionComponent = () => {
                     key={case_.id}
                     id={case_.id}
                     fullName={case_.entity.name}
-                    avatarUrl={case_.entity.avatarUrl}
                     createdAt={case_.createdAt}
-                    assignee={{
-                      id: case_.assignee?.id,
-                      fullName: `${case_.assignee?.firstName} ${case_.assignee?.lastName}`,
-                    }}
+                    assignee={
```

Comment:
Is there an importance for `null` here over optional chaining?

Reply:
It is done to avoid generating assignee entity where fields like fullName are generated as `undefined undefined` which eventually generates initials `UU` within Case.Item